src/router/index.js

```diff
@@ -888,7 +875,7 @@ const router = new Router({
         title: 'Room Detail',
         breadcrumb: [
           { name: 'Home', link: 'feeds' },
-          { name: 'Assignments', link: 'assignments' },
+          { name: 'Tasks', link: 'scoringAdmin' },
```

Comment:
Does student use `scoringAdmin` as well?

Reply:
Yes. scoringAdmin is actually tasks page, but the component is temporarily named scoringAdmin